src/script/interpreter.cpp

```diff
@@ -15,6 +15,9 @@
 
 typedef std::vector<unsigned char> valtype;
 
+static_assert(std::numeric_limits<int64_t>::max() == (int64_t)0x7FFFFFFFFFFFFFFF);
+static_assert(std::numeric_limits<int64_t>::min() == (int64_t)-0x8000000000000000);
```

Comment:
I'd rather have

    static_cast<uint64_t>(std::numeric_limits<int64_t>::max())==UINT64_C(0x7FFF'FFFF'FFFF'FFFF)
    static_cast<uint64_t>(std::numeric_limits<int64_t>::min())==UINT64_C(0x8000'0000'0000'0000)

Reply:
macros are incompatible with the ' separation. Changed to 
```
static_assert(static_cast<uint64_t>(std::numeric_limits<int64_t>::max()) == UINT64_C(0x7FFFFFFFFFFFFFFF));
static_assert(static_cast<uint64_t>(std::numeric_limits<int64_t>::min()) == UINT64_C(0x8000000000000000));
```